apps/music/test/marionette/lib/music.js

```diff
@@ -700,6 +723,25 @@ Music.prototype = {
     this.switchToMe();
   },
 
+  tapRepeat: function() {
+    this.waitForRatingOverlayHidden();
+    this.showSongInfo();
+
+    var frame = this.playerViewFrame;
+    assert.ok(frame);
+    this.client.switchToFrame(frame);
+
+    var cover = this.client.findElement(Music.Selector.playerCover);
+    this.client.switchToShadowRoot(cover);
+    assert.ok(cover);
```

Comment:
Yes, I agree. But iirc, I copied this part over from some other code and then adjusted it to my needs. I guess that code also needs to be fixed then.